examples/bigcommerce/README.md

```diff
@@ -0,0 +1,34 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```

Comment:
This file will be back filled with some marketing content from @apledger. No need for review

Reply:
Got it. Do we have an ETA on that?

Reply:
I don't. I will come up with something tomorrow morning based on what Alan made for thirdweb and update the readme/merge.